src/org/safs/rest/service/models/consumers/SafsrestAdapter.groovy

```diff
@@ -144,15 +143,11 @@ class SafsrestAdapter extends ClientPOJOAdapter {
         if (request.headers) {
             request.headers.each { entry -> optionList << /${CurlCommand.HEADER_OPTION}${entry.key}${CurlCommand.HEADER_FIELD_SEPARATOR}${entry.value}/ }
         }
-
-		if(auth2!=null /* TODO In what condition, we need to make request to authorization server. */){
```

Comment:
I will look more into this.  The rest adapter assumes the auth server protocol, host, and port are the same as the rest service being tested.